src/modules/page-anchor/page-anchor.component.ts

```diff
@@ -29,6 +29,7 @@ export class StachePageAnchorComponent implements OnInit, StacheNavLink {
   }
 
   public addHashToUrl(): void {
+    this.elementRef.nativeElement.scrollIntoView({behavior: 'smooth'});
```

Comment:
Spaces between curly braces. `{ behavior: 'smooth' }` (This needs to be added to the tsling.config, too. <-- working on it)